tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -920,6 +924,68 @@ def mirror_daemon_status(self, daemon):
             return 1
         return 0
 
+    def add_remove_osd(self, command, osd_nodes, config):
```

Comment:
Can we please avoid this method as part of the rbdmirror class as much as possible ?